WordPress/src/main/java/org/wordpress/android/viewmodel/main/WPMainActivityViewModel.kt

```diff
@@ -118,6 +118,16 @@ class WPMainActivityViewModel @Inject constructor(
     private fun loadMainActions(site: SiteModel?) {
         val actionsList = ArrayList<MainActionListItem>()
 
+        if (SiteUtils.supportsStoriesFeature(site)) {
```

Comment:
I think this block should be moved below the `Add new` action [here](https://github.com/wordpress-mobile/WordPress-Android/pull/14557/files#diff-e536c944784cffa9b67cd62bea37b9d8e9f34a59927b3a59edfd3495956c46a6R131-R138), otherwise the bottom sheet looks like this right now on this PR:

![Screenshot_20210430-104840](https://user-images.githubusercontent.com/6597771/116672297-658f6c80-a978-11eb-8922-992679e40e02.png)

Reply:
Oops, you're right! I was experimenting with some different solutions and somehow pushed the wrong version to the PR 🤦‍♂️ This was fixed in bf4a53b.

Reply:
Thanks for the fix. Code-wise I don't think it's good that both the title and the action use the same object called `CreateAction`. I think that probably caused this error. Could we maybe add a test or two for these buttons?

Reply:
Thanks for the change @renanferrari! 🙇